### DISCUSSION TITLE
Cannot rotate MediaElement on Android

### DISCUSSION THREAD

**Discussion by cagriy** [CommunityToolkit/Maui#1613] [Q&A]
Hello,

My app is permanently in portrait mode, and when I play a video, using MediaElement I'd like the video displayed 90 degrees rotated. i.e. just a landscape video player without changing the device orientation.

No problems in iOS, however, for Android, when I rotate the MediaElement, the frame of the video gets rotated, but, the actual content stays unchanged. A clearer description of the issue and sample videos can be seen here: https://github.com/CommunityToolkit/Maui/issues/1610

While the bug report is in progress, I wanted to check if anyone knows or may suggest a workaround ?

I tried to write a handler to rotate the `StyledPlayerView` directly, but it didn't improve anything (my knowledge is limited when it comes to handlers mappers etc, so I might have done something wrong).

3.0.1 MediaElement, same issue with .NET MAUI 7 and 8.

Many thanks

**Comment by CliffAgius** [comment]
Closing this discussion as there is an open issue to track #1610
